src/azure-cli/azure/cli/command_modules/aro/_params.py

```diff
@@ -55,9 +55,9 @@ def load_arguments(self, _):
                    help='Client secret of cluster service principal.',
                    validator=validate_client_secret(isCreate=True))
 
-        c.argument('install_version',
+        c.argument('version',
                    help='OpenShift version to use for cluster creation.',
-                   validator=validate_install_version_format)
+                   validator=validate_version_format)
```

Comment:
It's a breaking change, and it's not in the breaking change window  this month. You can use the `c.deprecate` function to hide `--install-version` option name until the breaking change window.
```
        c.argument('version', options_list=['--version', c.deprecate(target='--install-version', redirect='--version', hide=True)],
                   help='OpenShift version to use for cluster creation.',
                   validator=validate_version_format)
```

Reply:
Thank you for the pointer, I've pushed a change for this.